lib/presentation/my_plan/created_plans_tab_view.dart

```diff
@@ -1,14 +1,51 @@
 import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
 
-class CreatedPlanTabView extends StatelessWidget {
-  const CreatedPlanTabView({super.key});
+import '../../error_page.dart';
+import '../../i18n/strings.g.dart';
+import '../../utils/routes/app_router.dart';
+import '../components/loading_overlay.dart';
+import 'components/nondata.dart';
+import 'components/plan_list.dart';
+import 'my_plan_notifier.dart';
 
+class CreatedPlanTabView extends ConsumerWidget {
+  const CreatedPlanTabView({super.key});
   @override
-  Widget build(BuildContext context) {
-    return ListView(
-      children: const [
-        Text('created_plan'),
-      ],
+  Widget build(BuildContext context, WidgetRef ref) {
+    final asyncState = ref.watch(myPlanPageNotifierProvider);
+    final i18n = Translations.of(context);
+    final createItemi18n = i18n.myPlanPage.createdPlansItems;
+
+    return asyncState.when(
+      data: (state) {
+        final plans = state.createPlanList;
+        return plans.isEmpty
+            ? Nondata(
+                message: createItemi18n.nondata,
+                labelText: createItemi18n.createaplan,
+                onPressed: () async {
+                  await const CreatePlanPageRouteData().push<void>(context);
+                },
+              )
+            : RefreshIndicator(
+                onRefresh: () async {},
+                child: ListView(
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.all(16),
+                      child: PlanList(plans: plans),
+                    ),
+                  ],
+                ),
```

Comment:
PlanListがListViewウィジェットで囲われているはずなので、一番外側のListViewウィジェットは必要ないはず
```suggestion
                child: Padding(
                      padding: const EdgeInsets.all(16),
                      child: PlanList(plans: plans),
                    ),
```